### PR TITLE
Add cassandra-2.1.2-3.0.15-beta release notes.

### PR DESCRIPTION
**Pull request by mpereira**

### CODE REVIEW

File: frameworks/cassandra/docs/release-notes.md [hunk 1]
```diff
@@ -7,6 +7,19 @@ menuWeight: 120
 
 ---
 
+## Version 2.1.2-3.0.15-beta
+
+## NOTICE
+
+This is a beta release of the DC/OS Apache Cassandra framework. It contains multiple improvements as well as new features that are to be considered of beta quality. Do _not_ operate this version in production.
+
+### Improvements
```
Comment (elezar) [inline frameworks/cassandra/docs/release-notes.md]: In the other docs we use `Updates`?

Also, was there a Health-check change?

Comment (mpereira) [reply frameworks/cassandra/docs/release-notes.md]: I think we haven't been very consistent...

```
./frameworks/elastic/docs/release-notes.md:49:### Improvements
./frameworks/elastic/docs/release-notes.md:59:### Improvements
./frameworks/elastic/docs/release-notes.md:69:### Improvements
./frameworks/elastic/docs/release-notes.md:94:### Improvements/Features
./frameworks/template/docs/release-notes.md:11:### Improvements
./frameworks/kafka/docs/release-notes.md:42:### Improvements
./frameworks/kafka/docs/release-notes.md:56:### Improvements
./frameworks/cassandra/docs/release-notes.md:16:### Improvements
./frameworks/cassandra/docs/release-notes.md:54:### Improvements
./frameworks/cassandra/docs/release-notes.md:68:### Improvements
./frameworks/hdfs/docs/release-notes.md:51:### Improvements
./frameworks/hdfs/docs/release-notes.md:64:### Improvements
```

```
./frameworks/elastic/docs/release-notes.md:16:### Updates
./frameworks/elastic/docs/release-notes.md:35:### Updates
./frameworks/kafka/docs/release-notes.md:37:### Updates
./frameworks/cassandra/docs/release-notes.md:29:### Updates
./frameworks/cassandra/docs/release-notes.md:48:### Updates
./frameworks/hdfs/docs/release-notes.md:39:### Updates
```

> Also, was there a Health-check change?

Yes! I'll add it, thanks.

Comment (mpereira) [reply frameworks/cassandra/docs/release-notes.md]: 1c128bf0bc6b7f3d7940ee4c654ac256b83644f0

Comment (triclambert) [reply frameworks/cassandra/docs/release-notes.md]: IIRC...

Bug Fixes
Improvements
Breaking Changes

File: frameworks/cassandra/docs/release-notes.md [hunk 2]
```diff
@@ -7,6 +7,20 @@ menuWeight: 120
...
+- Remove the dependency on a docker image and allow the service to run as nobody in CentOS distros. [#2067](https://github.com/mesosphere/dcos-commons/pull/2067)
+- Moved scheduler health-checks from HTTP to [Mesos-level health checks](https://mesosphere.github.io/marathon/docs/health-checks.html#mesos-level-health-checks).
```
Comment (triclambert) [inline frameworks/cassandra/docs/release-notes.md]: How does this impact an operator?

Comment (mpereira) [reply frameworks/cassandra/docs/release-notes.md]: @gabrielhartmann might have more details, but from what I understand the biggest impact of switching to MESOS_HTTP is decreasing the number of false positives due to networking failures during health checks, and also moving health-checking overhead from the scheduler to the agents running service tasks.

**Pull request merged**